### PR TITLE
wlogout: Add at 1.2.2

### PR DESCRIPTION
**Pull request by Facundo-c-c**
**Summary**

Add at 1.2.2

Resolves https://github.com/getsolus/packages/issues/2765

**Test Plan**

tested and used on a Wayland session

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/w/wlogout/package.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+release    : 1
+source     :
+    - https://github.com/ArtsyMacaw/wlogout/archive/refs/tags/1.2.2.tar.gz : 4c9204bfa19c73f51176c94c67711f54f3e393301c0809c61ae379054060fa46
+homepage   : https://github.com/ArtsyMacaw/wlogout
+license    : MIT
+component  : desktop
+summary    : A wayland based logout menu
+description: |
+    wlogout is a logout menu for wayland environments
+builddeps  :
```
Comment (malfisya) [inline packages/w/wlogout/package.yml:12]: Lets do what the linter said

File: packages/w/wlogout/pspec_x86_64.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    </Source>
+    <Package>
+        <Name>wlogout</Name>
+        <Summary xml:lang="en">A wayland based logout menu</Summary>
+        <Description xml:lang="en">wlogout is a logout menu for wayland environments
+</Description>
+        <PartOf>desktop</PartOf>
+        <Files>
+            <Path fileType="config">/etc/wlogout/layout</Path>
+            <Path fileType="config">/etc/wlogout/style.css</Path>
```
Comment (malfisya) [inline packages/w/wlogout/pspec_x86_64.xml:24]: What does this file do? Can we move it to `/usr` ?

**Review by malfisya — CHANGES_REQUESTED** [review]
Let's use a better commit message:
```
package: Add at v1.0
```
Don't forget to edit your github PR message too

File: packages/w/wlogout/pspec_x86_64.xml (hunk 2 above)
Comment (Facundo-c-c) [reply packages/w/wlogout/pspec_x86_64.xml:24]: Just default config, move them into `/usr` means patch the code, could be it deleted and expect than users create them into their `/home`? or delete them and move them into `/.config` by default?

File: packages/w/wlogout/package.yml (hunk 1 above)
Comment (Facundo-c-c) [reply packages/w/wlogout/package.yml:12]: You mean, just use the summary description?

File: packages/w/wlogout/pspec_x86_64.xml (hunk 2 above)
Comment (malfisya) [reply packages/w/wlogout/pspec_x86_64.xml:24]: From staff discussion, this packages needs to be patched to support our stateless model. Default config files should be shipped in `/usr/share/defaults`. This is also hard requirement when we move to Serpent, as we have to deal with other 157 offending packages that also does this.

Comment (Facundo-c-c) [reply packages/w/wlogout/pspec_x86_64.xml:24]: okay, i will do it

File: packages/w/wlogout/files/0001-Make-wlogout-stateless.patch [hunk 3]
```diff
@@ -0,0 +1,55 @@
+From 6cd5181b92f067e40194ed598bc94d2d0e249b17 Mon Sep 17 00:00:00 2001
+From: Facundo Ciruzzi <ciruzzifacundo@gmail.com>
+Date: Wed, 26 Jun 2024 17:33:21 -0300
+Subject: [PATCH] create_patch
```
Comment (ReillyBrogan) [inline packages/w/wlogout/files/0001-Make-wlogout-stateless.patch:4]: Please change the subject to match what the file is named. Also, please generate this patch by using `git format-patch --zero-commit --no-signature`

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
Also please add a monitoring.yml

**Pull request merged**